packages/@okta/vuepress-site/docs/reference/api/factors/index.md

```diff
@@ -449,6 +449,147 @@ curl -v -X GET \
 ]
 ```
 
+### List YubiKey OTP Tokens
```

Comment:
This is an existing endpoint that I didn't see doc'd (the code change in 2020.07.0 only adds the single token GET and upload).  I wasn't sure if we wanted to add doc for this or not.  It also takes a number of optional request parameters: filter, limit, after, expand, forDownload, sortBy, sortOrder

Reply:
Why not add those in a `Request parameters` section?

Reply:
Added request parameters section and created https://oktainc.atlassian.net/browse/OKTA-308883 to add to the Postman collection